README.md

```diff
@@ -526,6 +526,30 @@ The **`deleteModule | dm`** task will delete a sub project, this task has one re
 
 Review the issues, we hear new ideas. Read more [Contributing](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing)
 
+## Analytics
```

Comment:
will be useful add the  analytics task with the shortcut to set the state of the project